dev/Deployment/DeploymentManager.cpp

```diff
@@ -357,36 +391,50 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implem
         return std::wstring{ path.get() };
     }
 
-    // Adds the package at the path using PackageManager.
+    // If registerHigherVersionPackage == false, Adds the current version package at the passed in path using PackageManager.
+    // If registerHigherVersionPackage == true, Registers the higher version package using the passed in path as manifest path and PackageManager.
```

Comment:
Yes, that's the intent. Use existing package if it's higher version than the current framework package version. Otherwise, just addpackageasync on the main/singleton package from the current framework package version. Ok, I will take your suggestion for the variable name - it definitely gives out more context.